src/appshell/internal/applicationuiactions.cpp

```diff
@@ -42,15 +42,15 @@ const UiActionList ApplicationUiActions::m_actions = {
              ),
     UiAction("about",
              mu::context::UiCtxAny,
-             QT_TRANSLATE_NOOP("action", "About...")
+             QT_TRANSLATE_NOOP("action", "About�")
```

Comment:
Just for you @Jojo-Schmitz

Reply:
D'oh, except now it looks like this 
![image](https://user-images.githubusercontent.com/5364225/130601308-c0b951a8-7656-4e8e-8760-67c94b51f246.png)

Reply:
Not good. I'm currently fighting with this too. Somehint about UTF8 vs. some Windowes codepage I guess

Reply:
It does work for editstyle.ui (in addition to make them untranslatable) and in notationuiactions.cpp though

Reply:
It's exactly that, for some reason notationuiactions.cpp gets "correctly" guessed as being UTF-8 whereas applicationuiactions.cpp doesn't.  I honestly have no idea why yet. I can put a UTF-8 encoding marker at the beginning but I don't know if all compilers on all platforms will recognise that correctly?

Reply:
They should, this explict marker is in some/many(?) files already I believe

Reply:
Visual Studio seems to have some internal heuristic to automagically detect this, apparently not really reliable though

Reply:
I think I've fixed it, even without the UTF-8 encoding marker, will push again

Reply:
Well I saved them *with* the marker then removed them with a binary editor and they seem OK now - but yes, probably better to have the markers there as long as all tools on all platforms know how to read them (including uncrustify etc.)

Reply:
Mind to do the same for diagnosticsactions.cpp, extensionsservice.cpp, editmidimappingmodel.cpp, instrumentsuiactions.cpp, languagesservice.cpp, projectuiactions.cpp?

Reply:
editstyle.ui and notationsuiactions.cpp need the ... change too, but not the UTF one

Reply:
I think I've changed quite enough for one PR! They're not really related and I don't have time to test them all.  notationuiactions.cpp should be fixed throughout.

Reply:
OK. I'll see what I can do to the rest

Reply:
Interestingly that question-mark in a diamond is how it shows in the notification email too, but not on github.com. Go figure.  If you're confident that the UTF-8 BOM works then we should add it all files (or at least, any that have non-ASCII characters).

Reply:
See #8918

Reply:
The 'trick' was to use vi(m) rather than the builtin editor from Visual Studio

Reply:
It's been merged, maybe you need (or want) to rebase